scripts/travis/build-all-platform-binaries.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Builds all platform binaries.
+set -e
+
+BRANCH=${BRANCH:?'missing BRANCH env var'}
+
+set -x
+
+if [[ $BRANCH == "master" ]]; then
```

Comment:
It's probably better to have an env var like `SKIP_UI`, which is set to `true` if the branch isn't master. This way, if I run this script locally, I can override the `SKIP_UI` env var even if I'm on a non-master branch.

Reply:
Yeah that's better. Made all such main scripts have a `SKIP_UI` option instead.